napi/index.js

```diff
@@ -0,0 +1,380 @@
+// prettier-ignore
+/* eslint-disable */
+// @ts-nocheck
+/* auto-generated by NAPI-RS */
+
+const { createRequire } = require('node:module')
```

Comment:
Please use ESM https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
I can try but I am not an expert on this and there could be bugs in NAPI when using the new esm option, because not many projects use it and it's not adopted much. If not strictly necessary I would avoid it.